lib/validation-schemas.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const { readFileSync } = require('fs');
+const { dirname, join } = require('path');
+const { runInNewContext } = require('vm');
+
+const moduleDir = dirname(__dirname);
+const schemasDir = join(moduleDir, 'schemas');
+const domainsPath = join(schemasDir, 'metaschema', 'domains.schema');
+const domainSchemaPath = join(schemasDir, 'metaschema', 'Domain.schema');
+const databaseFieldSchemaPath = join(
+  schemasDir, 'metaschema', 'DatabaseField.schema'
+);
+const { all: decorators } = require('./decorators');
+const schemas = [
+  ['domains', domainsPath],
+  ['DatabaseField', databaseFieldSchemaPath],
+  ['Domain', domainSchemaPath],
+].map(([name, path]) => [name, readFileSync(path)])
+  .map(([name, source]) => {
+    const sandbox = Object.assign({}, decorators);
+    sandbox.global = sandbox;
+    const code = `'use strict';\n(${source})`;
+    const options = {
+      timeout: 2000,
+      lineOffset: 1,
+    };
+    return [name, runInNewContext(code, sandbox, options)];
+  });
+const domains = schemas[0][1];
+const domainSchema = schemas[2][1];
+Object.assign(domains, domainSchema);
```

Comment:
File `Domain.schema` isn't valid category schema, it is described as domains schema, so I just assigned all domains from Domain.schema to domains

Reply:
We should make it a valid one, because of that we may need to create a `system_domains.schema` that will only be used for system schemas as `Domain.schema` and `DatabaseField.schema` if necessary.